### PR TITLE
commit from Dmytro

### PR DESCRIPTION
**Pull request by maliarovdmytro76**
my great commit

### CODE REVIEW

File: README.md
Comment (Ammars24) [inline README.md:1]: I agree

Comment (maliarovdmytro76) [reply README.md:1]: Ok.

**Review by maliarovdmytro76 — COMMENTED** [review]
alles gut.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # GitHub Study Project
 This is repository for researching GitHub tools and fuetures
+change drom Dmytro
```
Comment (Ammars24) [inline README.md:3]: I think this ok

**Comment by Ammars24** [comment]
my second  commit

**Pull request closed without merging**